node/Cargo.toml

```diff
@@ -10,7 +10,7 @@ anyhow = "1.0.92"
 async-trait = "0.1.83"
 axum = "0.7.9"
 borsh = { version = "1.5.1", features = ["derive"] }
-cait-sith = { git = "https://github.com/Near-One/cait-sith", rev = "27b9ae1ae394ad9d8f40e75a5fbab46242409165" }
+cait-sith = { git = "https://github.com/kuksag/cait-sith", rev = "b61d5969fb33b0c9eb4e41f9224cee41fe48a6b5", features = ["k256"] }
```

Comment:
let's not change this and wait until we have https://github.com/Near-One/cait-sith/pull/19 merged